.github/workflows/push-auto-generate.yml

```diff
@@ -0,0 +1,94 @@
+name: BRM / Automaticly Triggered BRM Generation
+
+on:
+  push:
+    branches-ignore:
+      - main
+    paths:
+      - modules/**.bicep
+
+jobs:
+  check-secret:
+    runs-on: ubuntu-latest
+    outputs:
+      my-key: ${{ steps.my-key.outputs.defined }}
+    steps:
+      - id: my-key
+        if: "${{ env.MY_KEY != '' }}"
+        run: echo "::set-output name=defined::true"
```

Comment:
`set-output` is [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/). Consider using the new syntax instead: `echo "{name}={value}" >> $GITHUB_OUTPUT`, like you did in the changed_folders step.

Reply:
```suggestion
        run: echo "name=true" >> $GITHUB_OUTPUT
```